app/components/molecules/section-name/section-name.styles.tsx

```diff
@@ -0,0 +1,32 @@
+import styled, {css} from 'styled-components';
+
+export const StyledSectionName = styled.div`
+  position: relative;
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  align-items: flex-start;
+  max-width: 300px;
+  max-height: 78px;
+`;
+
+export const LeadingIcon = styled.img`
+  width: 60px;
+  margin-right: 20px;
+  align-self: center;
+`;
+
+export const Label = styled.h1`
+  color: ${({theme}) => theme.colors.black};
+  font-weight: 400;
+  font-family: ${({theme}) => theme.fonts.title};
+  font-size: 32px;
+`;
+export const Border = styled.div`
```

Comment:
can this border be part of the label? do we have a different style at all?

Reply:
so I if use border into the label it will grow as the label grows itself. I'm trying to not use the <Hr /> component. Should I create another div that has the size of that line with its respective height and width?